databook/physiology/ephys/neuropixels-invalid-times.md

```diff
@@ -0,0 +1,6 @@
+## Invalid times in Neuropixels datasets
+In some sessions, there were minor problems with data acquisition that did not necessitate failing the entire experiment, but invalidated some of the data during small time periods. During these periods, channel data is recorded as `NaN`.
```

Comment:
I'd remove "minor" - they are minor enough that they don't necessitate failing the entire experiment, but they are "major" enough that we have to NaN the channel. Just say problems.

Reply:
As I understand it, more than one probe, and possibly all probes, can be affected at one time. From the white paper:


> Occasionally, there were problems with data acquisition that did not warrant failing the entire experiment, but did cause a portion of the data to be marked as invalid.
>
> Stimulus failures. Frames that remained on the screen for >3x the standard frame duration (50 ms) invalidated the stimulus for at least 1 s before and after the long frames. This occurred in 7 sessions, for a total of 312 s of invalid data across all probes (out of a possible 64,260 s).
>
> Data loss for one probe. The firmware buffer on an individual Neuropixels 3a probe sometimes overflowed, requiring that probe’s data acquisition to be re-started in the middle of an experiment. This occurred for 15 probes across 13 sessions, for a total of 8650 s of invalid data for individual probes (out of a possible 716,040 s).
>
> Gain fluctuations. There was an issue with Neuropixels 1.0 probes that caused the spike band gain of all probes to
simultaneously drop to 0, and return to normal over the course of ~1 s. Saturating events in the LFP band co-occurred
with these gain fluctuations. These fluctuations were detected in 2 sessions, for a total of 98.5 s of invalid data across all probes (out of a possible 18,360 s).

A stimulus failure or gain fluctuation would affect all probes, right? I'll adjust the writing to address that.